packages/camera/camera/example/ios/RunnerTests/CameraMethodChannelTests.m

```diff
@@ -0,0 +1,101 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera;
+@import XCTest;
+@import AVFoundation;
+#import <OCMock/OCMock.h>
+
+@interface FLTThreadSafeFlutterResult : NSObject
```

Comment:
To fix the build, try to make this an extension interface, like so (similar to how it is defined in the `FLTThreadSafeFlutterResult.m` file):
```suggestion
@interface FLTThreadSafeFlutterResult ()
```